content/articles/where-are-your-servers-located.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Where are your servers located?
-excerpt: Our Anycast network consists of <%= POPS.count %> points-of-presence around the world.
```

Comment:
I couldn't find a way to get erb to work inside the frontmatter. I believe it should be reserved only for static data. FWIW, this doesn't seem to work in our compiled site's metadata either:

<img width="688" alt="image" src="https://github.com/user-attachments/assets/4fba4409-9ec4-481a-87fd-96f55626bf9d" />

Reply:
😄